### PR TITLE
Updated top-level class documentation for Funcotator.

### LINKED ISSUE #4654: Funcotator - Update documentation
Documentation for Funcotator needs to be updated to be consistent with current functionality.

### PR DESCRIPTION
**Pull request by jonn-smith**
Fixes #4654

### CODE REVIEW

File: src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java [hunk 1]
```diff
@@ -172,6 +174,7 @@
  *   -R reference.fasta \
  *   -V input.vcf \
  *   -O output.vcf \
```
Comment (LeeTL1220) [inline src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java]: You sure the output file should still have a name of ".vcf" if the output format is MAF?

**Comment by jonn-smith** [comment]
It's supposed to ignore transcript version numbers, but it looks like right now it doesn't.

I'll add in an issue for it.

File: src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java (hunk 1 above)
Comment (jonn-smith) [reply src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java]: Good point.  Fixed!

**Comment by jonn-smith** [comment]
It's still accurate, but this info needs to be added to.  I'll merge it, but we'll need to do another one of these in relatively short order.

**Pull request merged**